pom.xml

```diff
@@ -57,14 +57,18 @@
             <id>intellectualsites snapshots</id>
             <url>https://mvn.intellectualsites.com/content/repositories/snapshots/</url>
         </repository>
+        <repository>
+            <id>papermc</id>
+            <url>https://repo.papermc.io/repository/maven-public/</url>
+        </repository>
```

Comment:
Seems unnecessary as the paper api isn't used?

Reply:
I have removed the papermc repository.